src/status_im2/common/contacts/views.cljs

```diff
@@ -0,0 +1,77 @@
+(ns status-im2.common.contacts.views
```

Comment:
i think its better to move to context, because it's a screen and functionality, not like a separate small component, common is more for atomic components or utils , so it's better to move to `status-im2.contexts.add-new-contact.`

Reply:
At some point we need to revisit what are Status bounded contexts. For the time being I think it's okay to not worry too much about this, but bounded contexts are not screens, they are higher level domain concepts, and designing the product around these concepts can be beneficial for many reasons.

But for now I think @flexsurfer's suggestion is a better, and practical recommendation.